misc/git.talon

```diff
@@ -39,6 +42,7 @@ git log: "git log "
 git log changes: "git log -c "
 git merge: "git merge "
 git merge <user.text>:"git merge {text}"
+git mergetool: "git mergetool\n"
```

Comment:
Should the spoken form be written as `merge tool`, for Dragon?

Reply:
Done, thanks :) I gather Dragon is less able to parse words it does not know yet, compared to Conformer?